app/assets/javascripts/govuk_publishing_components/analytics/track-click.js

```diff
@@ -24,24 +24,26 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   GemTrackClick.prototype.handleClick = function (target) {
     var options = { transport: 'beacon' }
-    var linkText
+    var linkUrl
 
     // if clicked element hasn't got the right attributes, look for a parent that matches
     if (!target.hasAttribute('data-track-category') && !target.hasAttribute('data-track-action')) {
-      linkText = target.textContent || target.innerText
+      linkUrl = target.getAttribute('href')
```

Comment:
Do we have an idea of the impact of this change (switching from using the label/text to using the href/url? I'm under the impression that `track-click` is rather widely used across GOV.UK

Reply:
Ah, I see it's only been [recently introduced](https://github.com/alphagov/govuk_publishing_components/pull/2263).

Reply:
It should have no impact, because this is a feature I added this morning, but got wrong - this PR corrects that.

My previous PR https://github.com/alphagov/govuk_publishing_components/pull/2263 added the ability to include the link text as the label if no label was specified. But it should have been the link href, not the text 🤦